vmdb/app/models/ems_refresh/save_inventory_cloud.rb

```diff
@@ -82,7 +82,7 @@ def save_flavors_inventory(ems, hashes, target = nil)
     target = ems if target.nil?
 
     ems.flavors(true)
-    deletes = if target.kind_of?(ExtManagementSystem)
+    deletes = if (target == ems)
```

Comment:
I did this but reverted back. Was worried it was too much of a change. (if only semantic)